app/views/admin/creator_settings/_form.html.erb

```diff
@@ -8,20 +8,24 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <%= label_tag :logo_svg, class: "crayons-field__label" do %>
+  <div class="crayons-field__label relative">
     Logo
     <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
     <p id="logo-subtitle" class="crayons-field__description">Ideally SVG, but PNG will work, too. Min size: 300x300px.</p>
-  <% end %>
-  <span class="block border-none">
-    <%= file_field_tag :logo_svg, class: "crayons-btn crayons-btn--secondary  w-100 ", role: "button", required: true %>
-  </span>
-
-  <% if ::Settings::General.logo_svg.present? %>
-    <div class="site-logo">
-      <%= ::Settings::General.logo_svg.html_safe %>
+  </div>
+  <div class="flex gap-2">
+    <div class="flex items-center">
+      <%= file_field_tag :logo_svg, class: "screen-reader-only", required: true, accept: ".svg,.png,image/png,image/svg+xml", data: { "max-file-size-mb": "25", action: "change->creator-settings#previewLogo" } %>
```

Comment:
Now only png and SVG files are permitted.

![image](https://user-images.githubusercontent.com/833231/141541069-2d61bc92-0141-43fd-abd1-c5106a6f6e1c.png)

Reply:
Any reason for NOT supporting GIF or JPG?

Reply:
We can. I thought the requirement was SVG or PNG.

Reply:
Yea, that probably wasn't very clear. I think we should enable all sensible format: SVG, PNG, JP(E)G and GIF. Obviously SVG is recommended, and PNG is, I guess 2nd recommended... :D

Reply:
I'd maybe suggest we don't allow GIF? It would be pretty bad for accessibility to have an animated logo when we don't currently have any settings to disable that. We already have (or had, at least) plans to disable this for user avatars - https://github.com/forem/rfcs/issues/125#issuecomment-786926070

Reply:
I spoke with @Ridhwana. @aitchiss and @ludwiczakpawel, I added JPG, but we're opting out of GIF for now to avoid the potential for someone uploading an animated GIF.